src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -529,43 +525,104 @@ public boolean verifyNewRankOptionsQuestionFormIsDisplayed() {
-        if (cal.get(Calendar.DATE) != 1) {
-            cal.add(Calendar.DATE, -1);       
 
-            // get month names
-            DateFormatSymbols dfs = new DateFormatSymbols();
-            String[] months = dfs.getMonths();        
+    public boolean isDatesOfPreviousCurrentAndNextMonthEnabled() {
+        return isDatesOfPreviousCurrentAndNextMonthEnabled(startDateBox) 
+               && isDatesOfPreviousCurrentAndNextMonthEnabled(endDateBox);
+    }
+
+    /**
+     * @param dateBox
+     * @return true if the dates of previous, current and next month are
+     *         enabled, otherwise false
+     */
+    private boolean isDatesOfPreviousCurrentAndNextMonthEnabled(WebElement dateBox) {
+
+        Calendar previousMonth = Calendar.getInstance();
+        previousMonth.add(Calendar.MONTH, -1);
 
-            // yesterday's day, month and year
-            String day = Integer.toString(cal.get(Calendar.DATE));
-            String month = months[cal.get(Calendar.MONTH)];
-            String year = Integer.toString(cal.get(Calendar.YEAR));
+        // Navigate to the previous month
+        if (!navigate(dateBox, previousMonth)) {
+            return false;
+        }
 
-            // go to previous month
-            while (!(getDatepickerMonth().equals(month) && getDatepickerYear().equals(year))) {
-                browser.driver.findElement(By.className("ui-datepicker-prev")).click();
+        // Check if the dates of previous, current and next month are enabled 
+        for (int i = 0; i < 3; i++) {
+
+            List<WebElement> dates = browser.driver.findElements(By.xpath("//div[@id='ui-datepicker-div']/table/tbody/tr/td"));
+
+            for (WebElement date : dates) {
+
+                boolean isDisabled = date.getAttribute("class").contains("ui-datepicker-unselectable ui-state-disabled");
+                boolean isFromOtherMonth = date.getAttribute("class").contains("ui-datepicker-other-month");
+
+                if (isDisabled && !isFromOtherMonth) {
+                    return false;
+                }
             }
 
-            // different months with the same day (e.g. first week or last week of the month) in the datepicker
-            List<WebElement> daysWithSameDayAsYesterday = browser.driver.findElements(By.xpath("//td[a[text() = \"" + day + "\"]]"));
-            
-            for (WebElement yesterday : daysWithSameDayAsYesterday) {
-                // yesterday is enabled
-                if (!yesterday.getAttribute("class").contains("ui-datepicker-unselectable ui-state-disabled")) {
+            // Navigate to the next month
+            browser.driver.findElement(By.className("ui-datepicker-next")).click();
+        }
+
+        return true;
+    }
+
+    /**
+     * Navigate the datepicker associated with {@code dateBox} to the specified {@code date}
+     * 
+     * @param dateBox
+     * @param date
```

Comment:
I think this header comment is good, but you should explain the parameter if you want to include the param in the javadoc.